com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Templates/ShaderPass.template

```diff
@@ -149,6 +149,7 @@ Pass
 
     // Includes
     $splice(PreGraphIncludes)
+	$splice(GraphIncludes)
```

Comment:
I noticed that the location for HDRP is before the pass structs and packing, should this be the case or should this match all the other templates (right before GraphFunctions)?

Reply:
It just has to be after the property declarations --  HDRP just happens to already be set up that way (they have their global includes in the template instead of being declared via the target/subshader includes).